packages/psss/src/components/AffectedSitesTab.js

```diff
@@ -63,15 +63,15 @@ export const AffectedSitesTab = ({ alert }) => {
     }
 
     return weeklyData.map(weekData => {
-      const { status, period, totalCases, percentageChange, sites: siteData } = weekData;
+      const { status, period, weeklyCases, percentageChange, sites: siteData } = weekData;
```

Comment:
We can change the property in the report to be `totalCases` instead. Personally I'm fine with calling it weekly cases so that's why I changed the front end part to make it easier